Caboodle/Vibration/Vibration.uwp.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Windows.Foundation.Metadata;
+using Windows.Phone.Devices.Notification;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Vibration
+    {
+        internal static bool IsSupported
+        {
+            get
+            {
+                if (ApiInformation.IsTypePresent("Windows.Phone.Devices.Notification.VibrationDevice"))
+                {
+                    return VibrationDevice.GetDefault() != null;
```

Comment:
Small refactoring would be to add:

```
internal static DefaultVibrationDevice => VibrationDevice.GetDefault();
```

This way you don't duplicate it 3 times.